kernel/src/main/scala/cats/kernel/Eq1.scala

```diff
@@ -0,0 +1,65 @@
+package cats.kernel
+
+import scala.annotation.implicitNotFound
+
+/**
+ * Lifting of the [[Eq]] class to unary type constructors.
+ *
+ * @see [[https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Functor-Classes.html#t:Eq1]]
+ */
+@implicitNotFound("Could not find an instance of Eq1 for ${F}")
```

Comment:
See #4060. I guess it was decided to get rid of the `@implicitNotFound` annotation unless it is absolutely necessary.

Reply:
👍 I'll drop it.

Reply:
dropped